saleor/graphql/discount/mutations.py

```diff
@@ -127,6 +141,9 @@ class VoucherInput(graphene.InputObjectType):
     products = graphene.List(
         graphene.ID, description="Products discounted by the voucher.", name="products"
     )
+    variants = graphene.List(
+        graphene.ID, description="Variants discounted by the voucher.", name="variants"
```

Comment:
We should use `saleor/graphql/core/descriptions.py::ADDED_IN_31 for all new fields/inputs

Reply:
Done (in all places I hope).